packages/myst-cli/src/process/mdast.ts

```diff
@@ -215,6 +215,7 @@ export async function transformMdast(
 
   if (execute) {
     const cachePath = path.join(session.buildPath(), 'execute');
+    console.log(JSON.stringify(frontmatter, null, 2));
```

Comment:
```suggestion
```

Reply:
:face_holding_back_tears: 

I officially cannot multitask.

Reply:
Thank you! Can you also add a changeset! :)